src/pymovements/dataset/dataset_files.py

```diff
@@ -354,6 +359,7 @@ def load_gaze_file(
                 add_columns=fileinfo_columns,
                 # column_schema_overrides is used for fileinfo_columns passed as add_columns.
                 column_schema_overrides=column_schema_overrides,
+                **load_function_kwargs,
```

Comment:
not added for `preprocessed` branch, as the `load_kwargs` are only relevant for the original source files (same reason why `definition` is also not forwarded there